content/docs/dataform_cli/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Dataform CLI
+priority: 4
+---
+
+The Dataform CLI is a submodule of the web application. It provides much of the functionality of the web app, but not the additional web services that are useful for larger teams.
```

Comment:
Instead of describing what it is not, can we describe what it is?Something like: 

The CLI enables you to initialize, compile test and run Dataform projects directly from your local machine or as part of other systems (e.g. Airflow)....

Reply:
Ah yes, that page wasn't actually visible because the index's aren't pages now. I'll move the sentence into "getting started" with the CLI